core/b2b/loader.tsx

```diff
@@ -4,11 +4,13 @@ import { auth } from '~/auth';
 
 import { ScriptDev } from './script-dev';
 import { ScriptProduction } from './script-production';
+import { ScriptProductionCustom } from './script-production-custom';
 
 const EnvironmentSchema = z.object({
   BIGCOMMERCE_STORE_HASH: z.string({ message: 'BIGCOMMERCE_STORE_HASH is required' }),
   BIGCOMMERCE_CHANNEL_ID: z.string({ message: 'BIGCOMMERCE_CHANNEL_ID is required' }),
   LOCAL_BUYER_PORTAL_HOST: z.string().url().optional(),
+  PROD_BUYER_PORTAL_URL: z.string().url().optional(),
```

Comment:
nitpick, but are we cool with calling these LOCAL_BUYER_PORTAL_**_HOST_** and PROD_BUYER_PORTAL_**_URL_**, even though they both get validated as URLs? Should we just default to the existing pattern of calling it a HOST?

Reply:
I'm cool with that.